devtools/gradle/gradle.properties

```diff
@@ -0,0 +1 @@
+version = 999-SNAPSHOT
```

Comment:
Hmm, is it related to the issue at hand?

This has consequences on the release process if we add a version in the middle of things.

Reply:
Oh, I didn't know that, sorry

Reply:
Ah, sorry, good catch. It's not related. I keep adding this file locally when I'm working on gradle issues.

Reply:
Removed it.